interop/.formatter.exs

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
how about we merge interop into benchmarking or vice-versa, so minimize the amount of auxiliary directories?